<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOManager</class>
 <widget class="QWidget" name="IOManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#IOManager{border-image:url(:/img/sky)}</string>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出入库管理系统</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>20</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>20</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>420</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="tv">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>541</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubmit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbisin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>是否为入库</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHint">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>串口连接/断开</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leChange">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>290</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbkey0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClean">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清除输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDele">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSum">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>总计金额：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbkeyDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbkeyUp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+1</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbChangeNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>替换数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbxx">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>直接更改数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbxx_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>替换数量</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbkeyBS">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>BS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
